### ISSUE TITLE
[Accessibility][Supporting the platform]: Coachmark bubble is not visible properly in High contrast mode black and white.

### ISSUE THREAD

**Issue by marygans** [OfficeDev/office-ui-fabric-react#10576]
### Describe the issue:

Breakout from Coachmark pass #8031

#### Expected behavior:
https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bec787de0-7a76-4b21-b593-fb839dfa9cd8%7D&action=default&uid=%7BEC787DE0-7A76-4B21-B593-FB839DFA9CD8%7D&ListItemId=1808&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble&cid=99df029b-fc11-49b3-8165-1c5a8a353e9e
